src/uebung_2_1/Pq_Formel.java

```diff
@@ -0,0 +1,20 @@
+package uebung_2_1;
+
+public class Pq_Formel {

```

Comment:
Unterstriche in Klassennamen sind möglich, sollten aber nicht verwendet werden, wenn man den üblichen Namenskonventionen folgen möchte.